spotinst/resource_spotinst_credential_aws.go

```diff
@@ -0,0 +1,105 @@
+package spotinst
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/client"
+	"github.com/spotinst/terraform-provider-spotinst/spotinst/credential_aws"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spotinst/spotinst-sdk-go/service/account/providers/aws"
+	"github.com/spotinst/terraform-provider-spotinst/spotinst/commons"
+)
+
+func resourceSpotinstCredentialAWS() *schema.Resource {
```

Comment:
change it to resourceSpotinstCredentialsAWS

Reply:
changed to resourceSpotinstCredentialsAWS